docs/src/pages/en/showcase.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Media Chrome Showcase
+description: Links to other resources for all things Media Chrome and the ideas behind Media Chrome
+layout: ../../layouts/MainLayout.astro
+---
+
+## Who's using Media Chrome
+
+Here are some projects and web applications that currently use Media Chrome:
+
+[Mux Player](https://mux.com/player). Mux's in-house video player uses Media Chrome under the hood. In addition to the linked product page, Mux Player is also [open source under Mux Elements](https://github.com/muxinc/elements/tree/main/packages/mux-player).
+
+<img src="/public/mux-player-media-chrome.png" width="500" alt="Mux Player built with Media Chrome" />
+
+Player by Caffeine.tv on [brobible.com](https://brobible.com/sports/article/mike-tyson-made-30-million-biting-evander-holyfield/)
+
+<img src="/public/caffeine-tv-media-chrome.png" width="500" alt="Caffeine TV player built with Media Chrome" />
```

Comment:
This one seems missing too

Reply:
<img width="804" alt="CleanShot 2022-12-13 at 12 43 29@2x" src="https://user-images.githubusercontent.com/1256071/207406313-80222e2f-56a0-48a2-8135-d7ab380bd2b2.png">